### PR TITLE
Added an image in the documentation of the grid_generator

### PR DESCRIPTION
**Pull request by blaisb**
Added an image to the documentation of the cylinder_shell mesh generator and
a small description of said image to illustrate the results of this generator.
I tried to make a figure that was both small and clear.

### CODE REVIEW

**Comment by bangerth** [comment]
Do you think you could add an entry to `doc/news/changes/minor` that explains your change?

File: include/deal.II/grid/grid_generator.h [hunk 1]
```diff
@@ -1031,6 +1031,13 @@ namespace GridGenerator
    * to the triangulation.
    *
    * @note The triangulation passed as argument needs to be empty when calling this function.
+   *
+   * @image html cylinder_shell.png
+   *
+   * In this picture, a cylinder shell of length 2, inner radius 0.5, outer
+   * radius 1. is shown. The default argument for n_radial_cells and
```
Comment (tjhei) [inline include/deal.II/grid/grid_generator.h]:
```suggestion
   * radius 1 is shown. The default argument for n_radial_cells and
```

File: doc/news/changes/minor/20190806BrunoBlais [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Added an image for the cylindrical shell grid generator and
+a small discussion on what this image illustrates this mesh
```
Comment (tjhei) [inline doc/news/changes/minor/20190806BrunoBlais]:
```suggestion
a small discussion on what this image illustrates.
```

**Comment by tjhei** [comment]
Sorry, one more grammar fix.

**Comment by tjhei** [comment]
documentation only, so we are good to go!

**Pull request merged**